core/src/main/java/org/frankframework/receivers/JavaListener.java

```diff
@@ -177,7 +178,7 @@ private Message processRequest(@Nonnull MessageWrapper<M> messageWrapper, @Nonnu
 					} catch (ListenerException e) {
 						// Message with error contains a String so does not need to be preserved.
 						// (Trying to preserve means dealing with extra IOException for which there is no reason here)
-						return handler.formatException(null, session, message, e);
+						return formatExceptionUsingErrorMessageFormatter(session, message, e);
```

Comment:
I think this is quite strange, this listener should always just throw the exception, it's up to the receiver to format it using the correct `ErrorMessageFormatter` if any. This captures the exception and prevents the `Receiver` from doing it's job.